### PR TITLE
Apply black formatter and flake8/mypy check for tests.

### PR DESCRIPTION
**Pull request by ueshin**
### Description

Applies black formatter and flake8/mypy check for tests.

Also removed spaces at the end of the line in the whole codebase.

### CODE REVIEW

File: tests/integration/incremental_on_schema_change/models/incremental_ignore_target.sql [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{{ 
-    config(materialized='table') 
+{{
+    config(materialized='table')
```
Comment (allisonwang-db) [inline tests/integration/incremental_on_schema_change/models/incremental_ignore_target.sql:2]: I guess the black formatter doesn't recognize the quotes in Jinja files.

**Review by allisonwang-db — APPROVED** [review]
Looks good!

File: tests/integration/incremental_on_schema_change/models/incremental_ignore_target.sql (hunk 1 above)
Comment (ueshin) [reply tests/integration/incremental_on_schema_change/models/incremental_ignore_target.sql:2]: Yes, those files are out of scope of black. We have to manually handle those for now.

**Comment by ueshin** [comment]
Thanks! merging.

**Pull request merged**